tools/cxbe/Main.cpp

```diff
@@ -20,12 +20,14 @@ int main(int argc, char *argv[])
     char szDumpFilename[OPTION_LEN + 1] = { 0 };
     char szXbeTitle[OPTION_LEN + 1] = "Untitled";
     char szXbeTitleID[OPTION_LEN + 1] = "";
+    char szXbeRegions[OPTION_LEN + 1] = "";
     char szMode[OPTION_LEN + 1] = "retail";
     char szLogo[OPTION_LEN + 1] = "";
     char szDebugPath[OPTION_LEN + 1] = "";
 
     bool bRetail;
     uint32 dwTitleId = 0xFFFF0002;
+    uint32 dwRegions;
```

Comment:
This needs to get initialized or lines 121+ will read undefined values.

Reply:
It does in line 141 under the `else`

Reply:
That's a different control flow path, I'm talking about 121+

Reply:
Ah ok I see
The fix was put in the wrong commit I think